### ISSUE TITLE
Camera being covered by Tabview

### ISSUE THREAD

**Issue by BMwanza** [NativeScript/nativescript-camera#128]
### If the demo apps cannot help and there is no issue for your problem, tell us about it
Please, ensure your title is less than 63 characters long and starts with a capital
letter.

### Which platform(s) does your issue occur on?
- iOS
- iOS 11.4
- Real Device: iPhone 6

### Please, provide the following version numbers that your issue occurs with:

- CLI: 4.2.3
- Cross-platform modules: (check the 'version' attribute in the
`node_modules/tns-core-modules/package.json` file in your project)
- Runtime(s): ios version: "version": "4.2.0"

- Plugin(s): Plug in version: "nativescript-camera": "^4.0.2"

### Please, tell us how to recreate the issue in as much detail as possible. 
Describe the steps to reproduce it.
I have created a nativescript app using the tabview template, when launching the camera, it does not take the full screen, part of it is being blocked by the tabview

![image](https://user-images.githubusercontent.com/26012815/46211310-d53ca700-c2f8-11e8-8498-2cac54ecaa67.png)

**Comment by tsonevn** [comment]
Hi @BMwanza,
Can you send us a sample project, which can be used for reproducing the problem?

**Comment by BMwanza** [comment]
Sure, here is a .zip of a sample project I just whipped up. Again I'm using the NativeScript tabview template and simply calling the Camera.takePicture() function within a page as the demo app I've attached shows.

Again, the bug seems to appear on physical iOS devices, it works well on Android both physical and real devices. Regarding iOS simulators, it is not possible to tell if the bug occurs given that the iOS simulators just launch the photos app when this function is called and not a real camera.

**Comment by tsonevn** [comment]
Hi @BMwanza,
Can you attach the project to the issue?

**Comment by BMwanza** [comment]
Sorry I thought it attatched, looks like the file is too big to be attatched to the issue, I'm going to make a git repo that you can just clone will that work?

**Comment by BMwanza** [comment]
> Hi @BMwanza,
> Can you attach the project to the issue?

**Comment by BMwanza** [comment]
@tsonevn Sorry for all the confusion, I also accidently closed this disucssion, but here is the project you can just clone it to run it
https://github.com/BMwanza/cameraBug

**Comment by chrisbekas** [comment]
Having this exact same issue on the device as described; iOS 12, latest {N} and camera plug-in.

**Comment by BMwanza** [comment]
@chrisbekas Hi Chris, on a side note, I actually messaged you on the NativeScript Slack workspace regarding another issue I saw you were having with the Tabview, if you have a chance please view my message on Slack

**Comment by BMwanza** [comment]
Hey @tsonevn how's this going, any progress? Please let me know if there is any other information I could provide to help assist with resolving this bug.

**Comment by tsonevn** [comment]
Hi @BMwanza,
The issue is still under review. We will provide the more info when we find how the problem can be solved. Also, any PR, which provides a fix, will be highly  appreciated

**Comment by BMwanza** [comment]
Hi @tsonevn,

Any update on this? :)

Or is there a way to work around the bug like setting the tabview's visibility to hidden/collpased? It's just a pressing issue because our Development team is looking to submit our app as soon as possible

**Comment by tsonevn** [comment]
Hi @BMwanza, 
The fix is already available in the master branch. We will publish a new version of the plugin soon.

**Comment by BMwanza** [comment]
Awesome!! Thanks so much @tsonevn and @lini on behalf of my entire dev team we really appreciate it!

**Comment by tsonevn** [comment]
HI all,
The new plugin version available in npm.
